rules/S6367/cfamily/metadata.json

```
@@ -0,0 +1,21 @@
+{
+  "title": "Thread local variables should not be used in coroutines",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "15min"
+  },
+  "tags": [
+    "confusing",
+    "since-c++20",
+    "suspicious",
+    "unpredictable" 
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6367",
+  "sqKey": "S6367",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown"
```

```suggestion
  "quickfix": "infeasible"
```
cf suggestion in https://github.com/SonarSource/sonar-cpp/pull/1639#discussion_r758805067